modules/monitoring-setting-up-metrics-collection.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * monitoring/monitoring-your-own-services.adoc
+
+[id="setting-up-metrics-collection_{context}"]
+= Setting up metrics collection
+
+To use the metrics exposed by your service, you need to configure OpenShift Monitoring to scrape metrics from the `/metrics` endpoint. You can do this using a ServiceMonitor, a custom resource definition (CRD) that specifies how a service should be monitored, or a PodMonitor, a CRD that specifies how a pod should be monitored. The former requires a Service object, while the latter does not, allowing Prometheus to directly scrape metrics from the metrics endpoint exposed by a Pod.
```

Comment:
@sichvoge can we reference prometheus-operator upstream documentation here so we can redirect users to have more detailed context? This documentation is a good starter, but as a user I would like to know more about the actual spec types/fields I can set. We could also link to the downstream documentation fork, i.e. to https://github.com/openshift/prometheus-operator/blob/release-4.3/Documentation/api.md.

Reply:
@s-urbaniak I think the link is a question to the support team too. @sferich888 , can we include this link?

Reply:
We can link to this, but we need to get a downstream fork going. 
@vikram-redhat where are we on automating our api docs? Would that solve this or are the API docs @s-urbaniak reference different?

Reply:
@sferich888 https://github.com/openshift/openshift-docs/pull/18929. @jboxman should be merging it today. I think these API docs are different. @jboxman are you able to confirm?

Reply:
@vikram-redhat is right, those docs in #18929 are unrelated. I take it that you @sferich888 approved linking to https://github.com/openshift/prometheus-operator/blob/release-4.3/Documentation/api.md , which I did in 319b232.

Reply:
@vikram-redhat is right, those docs in #18929 are unrelated. I take it that you @sferich888 approved linking to https://github.com/openshift/prometheus-operator/blob/release-4.3/Documentation/api.md , which I did in 319b232.

Reply:
@vikram-redhat @rh-max, well, theoretically that content actually _is_ in the API docs, but it's all in a single assembly. So it's not portable and not useful currently in this context. But someday, that's the ultimate goal.

Reply:
@jboxman Oh, OK. Still, the required scope is covered by https://github.com/openshift/prometheus-operator/blob/release-4.3/Documentation/api.md , so we are using that.